src/battlelist.cpp

```diff
@@ -79,4 +89,4 @@ bool BattleList_Iter::BattleExists(BattleList::battle_id_t const& id)
 	if (!m_battlelist)
 		throw std::logic_error("BattleList_Iter::BattleExists(): no battlelist");
 	return m_battlelist->m_battles.find(id) != m_battlelist->m_battles.end();
-}
+}
```

Comment:
revert please

Reply:
i can't see any change here so i don't know what to revert...

Reply:
newline was removed